assets/src/edit-story/elements/text/updateForResizeEvent.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { MIN_FONT_SIZE, MAX_FONT_SIZE } from '../../constants';
+import {
+  calculateFitTextFontSize,
+  calculateTextHeight,
+} from '../../utils/textMeasurements';
+import { dataPixels } from '../../units/dimensions';
+
+function updateForResizeEvent(element, newWidth, newHeight) {
+  const { width, height } = element;
+  const isResizingWidth = Math.abs(newWidth - width) >= 1;
+  const isResizingHeight = Math.abs(newHeight - height) >= 1;
```

Comment:
I was testing this a bit and for some reason, it seems like even when resizing the width only, the font size changes. console-logging shows that `newHeight - height` displays difference even though it's actually not being resizing. Didn't debug more, can you reproduce this?

![resizing](https://user-images.githubusercontent.com/3294597/75166514-3f42c100-5724-11ea-8a4b-7a3e99dd4f31.gif)

Reply:
I think this is what @pbakaus flagged in #368.

Reply:
Ah, looks like it, I'm a bit of out of date at this moment ;) Thanks!